todo_app/data/trello_items.py

```diff
@@ -0,0 +1,84 @@
+import os
+import requests
+import json
+
+from todo_app.classes.card import Card
+
+api_key = os.getenv('TRELLO_API_KEY')
+token = os.getenv('TRELLO_API_TOKEN')
+board_id = os.getenv('TRELLO_BOARD_ID')
+todo_list_id = os.getenv('TRELLO_TODO_LIST_ID')
+done_list_id = os.getenv('TRELLO_DONE_LIST_ID')
+
+def get_from_trello(url):
+    headers = {
+        "Accept": "application/json"
+    }
+    
+    auth_data = {
+        "key": api_key,
+        "token": token
+    }
+
+    response = requests.get(url, headers=headers, params=auth_data)
+
+    return response
+
+def post_to_trello(url, new_task_name):   
+    headers = {
+        "Accept": "application/json",
+        "Content-Type": "application/json"
+    }
+
+    payload = json.dumps({
+        "key": api_key,
+        "token": token,
+        "name": new_task_name,
+        "pos": "bottom"
+    })
+   
+    response = requests.post(url, headers=headers, data=payload)
+
+    return response
+
+def update_item_list(url, new_list_id):   
+    headers = {
+        "Accept": "application/json",
+        "Content-Type": "application/json"
+    }
+
+    payload = json.dumps({
+        "key": api_key,
+        "token": token,
+        "idList": new_list_id,
+        "pos": "bottom"
+    })
+   
+    return requests.put(url, headers=headers, data=payload)
+
+def get_items():  
+    reqUrl = "https://api.trello.com/1/boards/{0}/lists?cards=open&card_fields=id,name&fields=name".format(board_id)
```

Comment:
Personal preference, but I do prefer the f-string approach to interpolation over `.format` - mostly so I can easily see which arguments are added where (and I don't need to count them to check I have them in the right order):
```python
    reqUrl = f"https://api.trello.com/1/boards/{board_id}/lists?cards=open&card_fields=id,name&fields=name"
```